locations/spiders/lidl_de.py

```diff
@@ -1,84 +1,63 @@
     allowed_domains = ["lidl.de"]
     handle_httpstatus_list = [404]
-    start_urls = ["https://www.lidl.de/de/filialsuche/s940"]
-
-    def parse_hours(self, store_hours):
-        opening_hours = OpeningHours()
-        if store_hours is None:
-            return
-
-        for store_day in store_hours:
-            day, hours = store_day.split(" ")
-            match = re.match(r"(\d\d:\d\d)-(\d\d:\d\d)", hours)
-            if match:
-                opening_hours.add_range(
-                    day=DAY_MAPPING[day],
-                    open_time=match.group(1),
-                    close_time=match.group(2),
-                    time_format="%H:%M",
-                )
-
-        return opening_hours.as_opening_hours()
-
+    start_urls = ["https://www.lidl.de/f/"]
     def parse_details(self, response):
-        properties = {}
-        if response.status == 200:
-            json_data = re.search(r"salePoints = eval\((.*?)\);", response.text)
-            if json_data:
-                json_data = json.loads(json_data.groups(1)[0])
-                for store in json_data:
-                    if store["country"] == "DE":
-                        properties = {
-                            "country": store["country"],
-                            "ref": store["ID"],
-                            "name": store["name"],
-                            "street": store["STREET"],
-                            "postcode": store["ZIPCODE"],
-                            "city": store["CITY"],
-                            "lat": store["X"],
-                            "lon": store["Y"],
-                        }
-                        hours = self.parse_hours(store["parsedopeningTimes"]["regular"])
-                        if hours:
-                            properties["opening_hours"] = hours
 
-                        yield GeojsonPointItem(**properties)
+        lidlShops = response.css('.ret-o-store-detail')
+
+        for shop in lidlShops:
+            shopAddress = shop.css('.ret-o-store-detail__address::text').extract()
+            street = shopAddress[0]
+            postalCode = shopAddress[1].split()[0]
+            city = shopAddress[1].split()[1]
+
+            openingHours = shop.css('.ret-o-store-detail__opening-hours::text').extract()
+            shopOpeningHours = {}
+            for openingHour in openingHours:
+                if(openingHour.split()):
+                    day = openingHour.split()[0]
+                    hours = openingHour.split()[1]
+                    shopOpeningHours[day] = hours
+
+            services = response.css('.ret-o-store-detail__store-icon-wrapper')[0]
+            link = services.css('a::attr("href")').get()
+            coordinates = link.split('pos.')[1].split('_L')[0]
+            latitude = coordinates.split('_')[0]
+            longitude = coordinates.split('_')[1]
+
+            properties = {
+                "ref": latitude+longitude,
+                "country": "Germany",
```

Comment:
This should be "DE", and because it's constant on all entries it should go in `item_attributes`.